### ISSUE TITLE
Toast Notification focuses original element after disappearing

### ISSUE THREAD

**Issue by lukaw3d** [grommet/grommet#7115]
Changing focus after toast notification disappears is disruptive.

### Expected Behavior

Don't change focus when toast notification disappears, or make it configurable

### Actual Behavior

See steps to reproduce

### URL, screen shot, or Codepen exhibiting the issue

See steps to reproduce

### Steps to Reproduce

- https://v2.grommet.io/notification
- click "Open Toast Notification"
- click empty space
- scroll away, read docs
- after 8 seconds notification disappears and page scrolls back up to "Open Toast Notification" https://github.com/grommet/grommet/blob/9aee30310819f75e770857d213ac164483996ea7/src/js/components/Layer/Layer.js#L31 

### Your Environment

- Grommet version: 2.34.2
- Browser Name and version:
- Operating System and version (desktop or mobile):

**Comment by lukaw3d** [comment]
Yes, in "Steps to reproduce" it restores focus to "Open Toast Notification".

Which accessibility guidelines are you referring to?

Modal Dialog? https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/#:~:text=When%20a%20dialog%20closes
Alert? https://www.w3.org/WAI/ARIA/apg/patterns/alert/#:~:text=they%20do%20not%20affect%20keyboard%20focus

**Comment by taysea** [comment]
Thank you for clarifying. I would agree that "Toast" which functions like an alert should not change the focused element based on the linked guidance. Adding accessibility and bug labels to this.

**Comment by jcfilben** [comment]
Check what WCAG rule this. Potentially related to https://www.w3.org/WAI/WCAG22/Understanding/on-input.html
